### PR TITLE
Implement embedding multiple files

### PR DESCRIPTION
**Pull request by stapelberg**

### CODE REVIEW

**Comment by dsymonds** [comment]
Can you resolve the conflicts please? (and squash afterwards)

File: goembed.go [hunk 1]
```diff
@@ -66,14 +111,34 @@ import (
...
+
+{{ if gt (len .Args) 0 }}
+{{ range $idx, $var := .Args }}
+{{ $n := printf "%s_%d" $.VarName $idx }}
+	r, err = gzip.NewReader(bytes.NewReader({{ $n }}_gzip))
+	if err != nil {
+		panic(err)
+	}
+	{{ $n }}, err = ioutil.ReadAll(r)
+	r.Close()
```
Comment (stapelberg) [inline goembed.go]: Closing the `gzip.Reader` directly so that the resource overhead doesn’t pile up as the number of files grows.

**Comment by stapelberg** [comment]
Done. Now also correctly handling the -gzip flag.

File: goembed.go [hunk 2]
```diff
@@ -21,41 +22,85 @@ var (
 func main() {
 	flag.Parse()
 
-	raw, err := ioutil.ReadAll(os.Stdin)
+	fmt.Printf("package %s\n\n", *packageFlag)
+
+	if *gzipFlag {
+		if err := gzipPrologue.Execute(os.Stdout, struct {
```
Comment (dsymonds) [inline goembed.go]: the args are too big here. Pull the statement out of the `if`, and just use a map; the trivial loss of type safety doesn't matter here.

```go
err := gzipPrologue.Execute(os.Stdout, map[string]interface{}{
  "Args": ...,
  ...
})
if err != nil { log.Fatal(err) }
```

Comment (dsymonds) [inline goembed.go]: (other than that, LGTM)

Comment (stapelberg) [inline goembed.go]: Done.

**Pull request merged**